### ISSUE TITLE
PoB closes randomly while working on builds

### ISSUE THREAD

**Issue by superness** [PathOfBuildingCommunity/PathOfBuilding#3807]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### What is the expected behaviour/value?

PoB never closes without me initiating or confirming that it can close

### What is the actual behaviour/value?

When I'm using PoB it will randomly close. There is a brief window that appears that looks like it is performing an update but quickly goes away.

### How to reproduce the issue

Open PoB
Use it or leave it idle for a period (1hr)

### Build code

```shell
Don't have one - I was working on a new build and it closed itself and deleted all my work!
```

**Comment by Lothrik** [comment]
This is probably a duplicate of #3733, as the trigger conditions required for that crash are very easy to accidentally reproduce while creating a new build.

**Comment by Lothrik** [comment]
This should be fixed by #4530.
